KNOWN_ISSUES.md

```diff
@@ -0,0 +1,95 @@
+# Swift for TensorFlow Known Issues
+
+This is a curated list of known Swift for TensorFlow issues. With every release,
+new issues are added and resolved issues are updated.
+
+Please see the [JIRA issue tracker](https://bugs.swift.org/projects/TF/issues)
+for a full list of known issues.
+
+## Version 0.3
+
+### TensorFlow Library
+
+* [ ] `Tensor` advanced indexing and striding are not supported on GPU.
```

Comment:
Is there an API link?

Reply:
They are actually supported. What's not supported on GPU are the subscript setters only (i.e., assignment).